pyramid_zipkin/zipkin.py

```diff
@@ -32,6 +33,8 @@ def tween(request):
                                   request) as context:
             response = handler(request)
             context.response_status_code = response.status_code
+            context.binary_annotations_dict = get_binary_annotations(request)
```

Comment:
how about also provide `response` to the callback as we have it by using post call?